### DISCUSSION TITLE
Custom verifyRequest page tries to open auth verify-request endpoint

### DISCUSSION THREAD

**Discussion by geekyayush** [nextauthjs/next-auth#7623] [Help]
### Environment

  System:
    OS: Windows 10 10.0.22621
    CPU: (16) x64 12th Gen Intel(R) Core(TM) i5-12500H
    Memory: 1.02 GB / 15.63 GB
  Binaries:
    Node: 18.16.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - C:\Program Files\nodejs\yarn.CMD
    npm: 9.5.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22621.1702.0), Chromium (113.0.1774.42)
    Internet Explorer: 11.0.22621.1

### Reproduction URL

https://github.com/nextauthjs/next-auth-example

### Describe the issue

I have defined a custom verifyRequest page in `AuthConfig` like this:
```
 verifyRequest: "/verify-request",
```
![image](https://github.com/nextauthjs/next-auth/assets/25874080/ce9b5259-2e4e-4fe6-9c75-8cb96547f14b)

But after trying to login via magic link, it opens up the following URL in client side:
```
http://localhost:3000/api/auth/verify-request?provider=email&type=email
```
![image](https://github.com/nextauthjs/next-auth/assets/25874080/f248b32c-a457-4dbb-8e9b-52f0905130be)

Could it be because I am using a combination of `auth/core` and `next-auth`?
auth/core in edge function and `next-auth/react` in the frontend

### How to reproduce

- Create a new custom page called `verify-request.tsx or jsx` in pages directory.
- Pass `verifyRequest: "/verify-request"` in pages property of `AuthConfig`.
- Login via Email provider

### Expected behavior

It should redirect to the following URL:
```
http://localhost:3000/verify-request
```

**Comment by geekyayush** [comment]
Any help please?

**Comment by geekyayush** [comment]
any answer please?

**Comment by oming** [comment]
same problem....

**Comment by tconroy** [comment]
+1, same problem. Anyone find a solution?

**Comment by Chelny** [comment]
Same issue here
